servus/uint128_t.cpp

```diff
@@ -92,8 +106,8 @@ uint128_t make_UUID()
         // chances of collissions up to a noticeable level.
 #ifndef _MSC_VER
         // http://stackoverflow.com/questions/14711263/c11-stdmutex-in-visual-studio-2012-deadlock-when-locked-from-dllmain
-        static std::mutex mutex;
-        std::unique_lock< std::mutex > lock( mutex );
+        static lck::mutex mutex;
+        lck::unique_lock< lck::mutex > lock( mutex );
```

Comment:
Off-topic: I'm not convinced that removing the locking in MSVC is the way to solve the loader deadlock issue, it makes the function non-reentrant on Windows, which is unexpected behaviour from a free function like this and it's not documented . If planets align, it may lead to very hard to trace runtime crashes. Isn't it possible to add an if statement to detect when this code is being called from DDLMain()?